bson/primitive/primitive.go

```diff
@@ -37,7 +37,17 @@ type DateTime int64
 
 // MarshalJSON marshal to time type
 func (d DateTime) MarshalJSON() ([]byte, error) {
-	return json.Marshal(time.Unix(int64(d)/1000, int64(d)%1000*1000000))
+	return json.Marshal(d.GetTime())
+}
+
+// GetTime returns the date as a time type.
+func (d DateTime) GetTime() time.Time {
```

Comment:
Since Go doesn't have getters, let's just call this `Time` or `ConvertToTime`.

Reply:
Renamed to `Time`